layers/+spacemacs/spacemacs-defaults/funcs.el

```diff
@@ -594,7 +599,11 @@ If ARG is not nil, assume yes for default."
 (defun spacemacs/delete-current-buffer-file-yes ()
   "Removes file connected to current buffer and kills buffer with assume yes."
   (interactive)
-  (funcall #'spacemacs/delete-current-buffer-file t))
+  (funcall #'spacemacs/delete-current-buffer-file
+           spacemacs-delete-current-buffer-file-arbitrary)
+  (unless spacemacs-delete-current-buffer-file-arbitrary
+    (message "Customer the `spacemacs-delete-current-buffer-file-arbitrary' \
+to delete buffer and file without confirmation.")))
```

Comment:
Please remove the warning if the legacy behaviour is wished. In this case its a conf decision and we should not complain about it.